### PR TITLE
[occm] Move CCM rbac to manifests/

### PR DESCRIPTION
**Pull request by ramineni**
**What this PR does / why we need it**:
This PR moves the required RBAC for OCCM to new location manifests/controller-manager 
And also delete the rbac for pvl-controller, which is not required.

/kind cleanup 

**Which issue this PR fixes(if applicable)**:
fixes #

**Special notes for reviewers**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: manifests/controller-manager/cloud-controller-manager-roles.yaml [hunk 1]
```diff
@@ -10,8 +10,10 @@ items:
     resources:
     - leases
     verbs:
-    - get
     - create
+    - get
+    - list
```
Comment (jichenjc) [inline manifests/controller-manager/cloud-controller-manager-roles.yaml:15]: THe commit message said it's move from somewhere ? so where its original place and why not remove them?
or if we add those list and watch, what's the reason behind it?

**Comment by ramineni** [comment]
@lingxiankong @jichenjc PTAL

File: manifests/controller-manager/cloud-controller-manager-roles.yaml (hunk 1 above)
Comment (lingxiankong) [reply manifests/controller-manager/cloud-controller-manager-roles.yaml]: @jichenjc Is your concern resolved?

**Pull request merged**